server/base/src/main/java/org/apache/accumulo/server/fs/PerTableVolumeChooser.java

```diff
@@ -45,19 +39,15 @@
   private final ConcurrentHashMap<ChooserScope,VolumeChooser> scopeSpecificChooserCache = new ConcurrentHashMap<>();
   private final RandomVolumeChooser randomChooser = new RandomVolumeChooser();
 
-  // TODO has to be lazily initialized currently because of the reliance on HdfsZooInstance. see
-  // ACCUMULO-3411
-  private volatile ServerConfigurationFactory lazyConfFactory = null;
+  // TODO get old property from 1.9 table.volume.chooser??
```

Comment:
As long as it fails cleanly if only the old props are set I am ok with not handling it.  I would not want the system to silently proceed assigning volumes and ignoring the old props.

Reply:
I looked at the code and it seems like if you configure the PerTableVolumeChooser with the old props that it will throw an exception when it does not find the new props.